pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[tool.black]
```

Comment:
if we could move as many of the python project files into a subdirectory as possible that would make me happy

Reply:
Sure, I explored this at first but there were a few things that looked to the root directory for configs (the big one is black which is the code formatter for python) and the work arounds were like, "create a bash script that automatically adds the --config-file /some/path/to/config". My main idea was to go for easy to run w/o having someone remember a bunch of command line flags for config files.

Additionally, I've worked in a number of projects that use python and js. I don't think I've ever been like, "let me shove the node config stuff into another folder b/c it's not python." The root allows one to see the files necessary to run the project. However, I do understand the need to keep the root clean and only have files necessary for the project's runtime. An example of python and js living happily together is here: https://github.com/openstax/research-eGrader

I'll try to get things working where python is hidden. That may require more time investment than I have this week so I may not finish this soon as next week I have other things I need to work on.

Reply:
oh hypothesis is another example that has js stuff in the root along with the python: https://github.com/hypothesis/h

Reply:
yea, again, not opposed. the idea is that we would be working in the py tests regularly (for almost every feature developed probably) and i'm not trying to make the py code second class, even for the ts files i push them down the tree as much as i can

Reply:
I was able to move most of the files into the tests directory. The only one I wasn't able to is the config for pre-commit.